utils/tree/Makefile

```diff
@@ -8,12 +8,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=tree
-PKG_RELEASE:=1
 PKG_VERSION:=2.0.2
-PKG_SOURCE_URL:=http://mama.indstate.edu/users/ice/tree/src
-PKG_HASH:=7d693a1d88d3c4e70a73e03b8dbbdc12c2945d482647494f2f5bd83a479eeeaf
+PKG_RELEASE:=2
```

Comment:
Would it be possible to use autorelease? :)

Reply:
I initially converted this to autorelease but found that it did not increment the PKG_RELEASE number.  I will switch it over and force push.